app/controllers/movies_controller.rb

```diff
@@ -4,27 +4,21 @@ class MoviesController < ApplicationController
   # GET /movies
   # GET /movies.json
   def index
-
     @movies = Movie.all
-   respond_to do |format|
+
+    respond_to do |format|
       format.html { render :index }
       format.json { render json: @movies, each_serializer: MovieSerializer }
     end
   end
-  # sort actions
-  def rating
-    @movies = Movie.rated.by_rating
-  end
-
-  def unrated
-    @movies = Movie.unrated
-  end
 
-  def latest
-    @movies = Movie.latest
+  # sort actions
+  def sort
```

Comment:
`rating`, `unrated` and `latest` are kind of `sorting`, so I create a sort action instead.